cdap-ui/app/cdap/components/DataPrepConnections/DatabaseConnection/DatabaseConnection.scss

```diff
@@ -235,7 +235,7 @@ $db-sprite-url: url('/cdap_assets/img/db-sprite.png');
   @include db-image(7);
 }
 
-.db-mssql6 {
+.db-sqlserver42 {
   @include db-image(8);
```

Comment:
so the issue here that depending on the version that the user has deployed, it may also be 41 (that is still avail for download at MS and I found it on Matt's cluster, so it does not seem to uncommon). I think we really ahve t omake it clear that we expect version 42, perhaps in the text in the UI, or in a tooltip, and if we find something else, we should make that clear in the error message

Reply:
Yes, that is true across this jdbc drivers in dataprep. This fix is just to make this name consistent with what we have in `Hub`

Reply:
ok as a quick fix.